akka-typed/src/main/scala/akka/typed/internal/EventStreamImpl.scala

```diff
@@ -0,0 +1,332 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com/>
+ */
+package akka.typed
+package internal
+
+import akka.{ actor ⇒ a, event ⇒ e }
+import java.util.concurrent.atomic.AtomicReference
+import scala.annotation.tailrec
+import scala.concurrent.{ Await, Promise }
+import akka.util.{ ReentrantGuard, Subclassification, SubclassifiedIndex }
+import scala.collection.immutable
+import java.util.concurrent.TimeoutException
+import akka.util.Timeout
+
+/**
+ * An Akka EventStream is a pub-sub stream of events both system and user generated,
```

Comment:
Mention in docs here that it's local. One more place to mention => less confused people.